src/DIRAC/Resources/Catalog/PoolXMLCatalog.py

```diff
@@ -162,15 +162,13 @@ def __init__(self, xmlfile=''):
       if isinstance(xmlfile, list):
         for xmlf in xmlfile:
           try:
-            _sfile = file(xmlf, 'r')
```

Comment:
I'm not convinced this line is needed but I'm not sure. Perhaps it's supposed to be deleting + recreating the file if it already exists? i.e.

```python
with open(xmlf):
   pass
```

Same applies slightly lower down.